src/util.ts

```diff
@@ -194,7 +194,7 @@ class Util {
      */
     public logDebug(...args) {
         args = Array.isArray(args) ? args : [];
-        let timestamp = dateFormat(new Date(), 'HH:mm:ss.l ');
+        let timestamp = `[${dateFormat(new Date(), 'HH:mm:ss.l')}]`;
```

Comment:
Just a nice enhancement to the debug log output to make the timestamps stand out.